changelog.md

```diff
@@ -67,7 +67,9 @@
 - two poorly documented and not used modules (`subexes`, `scgi`) were moved to
   graveyard (they are available as Nimble packages)
 
-
+- `gorge`, `staticExec` now raise AssertError if exitCode is not 0 instead of
```

Comment:
Assertions aren't catchable, I think we want something that is catchable to be raised.

Reply:
but that's at compile time; user can always use the cleaner `gorgeEx` variant which returns exitCode and handle accordingly so there's not loss of usability.

In any case, catching doesn't work:
```
static:
  try:
    let a = gorge("D20190116T211842")
  except OSError as e:
    echo "caught"
```

gives:
```
Error: cannot evaluate at compile time: currException
```